string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,30 @@
+/* exported isVowel */
+function isVowel(character) {
+  if (
+    character === 'a' ||
+    character === 'e' ||
+    character === 'i' ||
+    character === 'o' ||
+    character === 'u' ||
+    character === 'A' ||
+    character === 'E' ||
+    character === 'I' ||
+    character === 'O' ||
+    character === 'U'
+  ) {
+    return true;
+  } else if (
+    character !== 'a' ||
+    character !== 'e' ||
+    character !== 'i' ||
+    character !== 'o' ||
+    character !== 'u' ||
+    character !== 'A' ||
+    character !== 'E' ||
+    character !== 'I' ||
+    character !== 'O' ||
+    character !== 'U'
+  ) {
```

Comment:
You can just do `else`. No need to do all these checks again. You already did the check in your original `if` statement, so if it didn't pass, then you can just return `false` inside an `else` statement